dist/js-components.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+    <head>
+    <meta charset="utf-8">
+    <title>Mapzen Styleguide</title>
+    <meta name="description" content="Mapzen Styleguide">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="styles/styleguide.css">
+    <link rel="stylesheet" type="text/css" href="./lib/X-rayHTML.css">
+    <link rel="stylesheet" type="text/css" href="./lib/prism.css">
+    <link rel="stylesheet" type="text/css" href="styles/styleguidepage.css">
+</head>
```

Comment:
we have extraced version of head in `includes` folder, so that we can convinently manage head!